### PR TITLE
KAFKA-8609: add rebalance-total-time

### PR DESCRIPTION
**Pull request by vvcephei**
In addition to the existing metrics added in KAFKA-8609, add the total (cumulative) time spent during rebalances.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by vvcephei** [comment]
Hey @guozhangwang , I was trying to use the rebalance metrics you added in KAFKA-8609, and I struggled a bit getting the information I needed. What I wanted to know is what the total amount of time spent in rebalances was. With the existing metrics, I had to multiply the (sampled) average rebalance time by the (cumulative) total number of rebalances. Unfortunately, the resulting number is not really sensible, since one component is subject to decay and the other is not.

I realized the easiest thing for me would just be to add a new cumulative rebalance time metric, and I think it actually makes sense as an addition to the metrics from KAFKA-8609. A lot of my metrics experience is with Datadog, and in that system, you could use this metric to graph both when rebalances happen and how long each instance spent during that rebalance. Also, we're currently providing a rebalances-per-hour metric, but given this proposed metric, you could easily graph the time spent in rebalance in the last second, hour, day, or whatever. There are probably other uses, but my pitch is essentially that I think this would be a useful addition to piggy-back into the release.

File: clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java [hunk 1]
```diff
@@ -1066,6 +1067,13 @@ public GroupCoordinatorMetrics(Metrics metrics, String metricGrpPrefix) {
                         "several failed re-trials until it succeeded"),
                 new CumulativeCount()
             );
+            this.successfulRebalanceSensor.add(
+                metrics.metricName("rebalance-total-time",
```
Comment (guozhangwang) [inline clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java]: I think this is a good to add, about the metric name, wdyt about `rebalance-latency-total` or `rebalance-latency-cumulated-total` instead? I felt it is more consistent with other namings.

**Review by guozhangwang — COMMENTED** [review]
This LGTM, I can update the KIP once we've agreed on the metrics name.

**Comment by guozhangwang** [comment]
@vvcephei please also augment the existing unit test for the added metric as well :) since Jenkins is in a mood right now let's just verify unit tests locally.

**Comment by vvcephei** [comment]
Thanks for the review, @guozhangwang . I agree, `rebalance-latency-total` is a better name. I also updated the test.

**Comment by vvcephei** [comment]
Retest this, please.

**Comment by guozhangwang** [comment]
LGTM. Will update KIP-429 accordingly after merging.

**Pull request merged**